### PR TITLE
Update the "getting started" docs to use migrations.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Issue #2230.

### CODE REVIEW

File: docs/tutorial/queries.rst [hunk 1]
```diff
@@ -152,127 +164,163 @@ there in the ``actors`` by using a :ref:`computable
...
+    'default::Person' optional? [y,n,l,c,b,s,q,?]
+    y
+    Created movie_schema/migrations/00002.edgeql, id:
+    m15dogciav42crguwkslmjoucq5ieg25hcgk7ewxfmcmzf7othksda
+
+Third and final step in this sequence is to apply the migration:
+
+.. code-block:: bash
+
+    $ edgedb migrate --schema-dir movie_schema
```
Comment (1st1) [inline docs/tutorial/queries.rst:229]: `--schema-dir` -- I don't think we need this in the tutorial. We should use the default location.

Comment (vpetrovykh) [reply docs/tutorial/queries.rst:229]: You're right that we can just use the default for this tutorial.

I wonder if it's still worth mentioning that the directory can be specified via this option, just to reassure a newcomer that they still can pick some sensible location for their schema in the project.

Comment (1st1) [reply docs/tutorial/queries.rst]: > I wonder if it's still worth mentioning that the directory can be specified via this option, just to reassure a newcomer that they still can pick some sensible location for their schema in the project.

I don't think we should. We can't cover all side questions that readers might have -- the tutorial would become hard to read. So let's keep it as simple as possible and to the point.

**Comment by Dhghomon** [comment]
Just looked over the whole thing and only found one small typo - just delete **the** here:

questions to make sure that **the** EdgeDB correctly inferred the changes:
questions to make sure that EdgeDB correctly inferred the changes:

**Comment by vpetrovykh** [comment]
I've cleaned up the getting started section and now I've also updated the GraphQL sections so that they use the `extension` syntax.

File: docs/tutorial/install.rst [hunk 2]
```diff
@@ -31,7 +32,7 @@ Then, let's install and configure the EdgeDB server:
 .. code-block:: bash
 
     $ edgedb server install
-    $ edgedb server init
+    $ edgedb server init default
```
Comment (elprans) [inline docs/tutorial/install.rst:35]: Perhaps use something like `myapp` here to highlight that the choice of the instance name is arbitrary and up to the user.  `default` might be interpreted like something you _have_ to use, or even as some weird argument to `init` ("init with defaults or something").

File: docs/tutorial/queries.rst [hunk 3]
```diff
@@ -4,77 +4,82 @@
...
 
 After the database and schema are set up, we can add some actual data.
-For example, let's add "Blade Runner 2049" to the database. It's possible to
-add movie, director and actor data all at once:
+For this tutorial we will use the command-line REPL tool to do that,
+so let's start it up:
+
+.. code-block:: bash
+
+    $ edgedb
```
Comment (elprans) [inline docs/tutorial/queries.rst:12]: This should be `edgedb -I myapp` (or whatever instance name we choose for the tutorial).

**Comment by vpetrovykh** [comment]
I rebased the getting started section on top of the #2263. The `-I tutorial` is now used consistently.

**Review by elprans — APPROVED** [review]

**Pull request merged**